### PR TITLE
tag v1.1.0-rc.2

### PR DESCRIPTION
**Pull request by vbatts**
## to tag

this is would mean tagging 49af349 as `v1.1.0-rc.2` (now with more SemVer)

## Changes

1880a43 modify range check to not need bytes prefix
72a5455 Use artifactType from image manifest
b077648 Allow missing bytes= prefix
6520ed9 remove bytes= prefix on range header
18fdcbd fail CI if conformance fails
a5582e3 expect units in range header
f0122f6 move location check, allow missing bytes= prefix
bea9b6f Automate conformance testing on PR using zot
c6400af Upgrade to reggie v0.6.1 to fix issue with Accept header
fa4a69a skip TLS verification during conformance tests
27ae6c2 conformance: make blob mount and upload checks more strict
fc2e963 whitespace lol
a1d7ad1 Specify Warning header usage
851ea52 Add OCI-Chunk-Min-Length header
b962ef8 Update Jon Johnson's email
ab44270 Updating go modules in conformance
83a61e7 Bump golang.org/x/net in /conformance
6c52520 Bump golang.org/x/text from 0.3.6 to 0.3.8 in /conformance
ab32a6f spec and README: reduce to just image-manifest
e66aadc Update spec.md
5025143 OCI-Subject: <subject digest>
4e77264 Add Go 1.20 support
16cc054 Specify subject-processed response header
d06de52 Specify filters-applied response header
af2e081 Fix broken link to image indices in spec.
a330837 Add a patch status to recover failed requests
315c7e0 conformance.test01Pull: Check client errors before using response object
bd87eee docs: Update release process docs with checklist
95bed23 version: switch back to -dev

### CODE REVIEW

File: specs-go/version.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ const (
 	VersionPatch = 0
 
 	// VersionDev indicates development branch. Releases will be empty string.
-	VersionDev = "-dev"
+	VersionDev = "-rc.2-dev"
```
Comment (jdolitsky) [inline specs-go/version.go:28]: should be just `-rc2`:
```suggestion
	VersionDev = "-rc2"
```

Comment (jdolitsky) [reply specs-go/version.go:28]: or `-rc.2` (no dev). Right?

**Comment by jdolitsky** [comment]
See also https://github.com/opencontainers/image-spec/pull/1049

File: specs-go/version.go (hunk 1 above)
Comment (vbatts) [reply specs-go/version.go:28]: well, the idea is that `main` always stays with the `-dev` suffix, and we tag the commit in the middle

Comment (jdolitsky) [reply specs-go/version.go:28]: right but why `"-rc.2-dev"` and not `"-rc.2"` ?

Comment (jdolitsky) [reply specs-go/version.go:28]: going off of https://github.com/opencontainers/image-spec/pull/1049

**Review by jzelinskie — APPROVED** [review]

File: specs-go/version.go (hunk 1 above)
Comment (jdolitsky) [reply specs-go/version.go:28]: er, let me rephrase - why `"-rc.2-dev"` and not `"-dev"` ? (for the second commit)

**Review by jdolitsky — APPROVED** [review]

File: specs-go/version.go (hunk 1 above)
Comment (vbatts) [reply specs-go/version.go:28]: yeah ok fair. I can change it to just `-dev`

**Review by dmcgowan — APPROVED** [review]

**Review by sudo-bmitch — APPROVED** [review]
I updated the change list to include #401. It is in the git history for this, so it looks like it was just a race condition in writing the change list.

**Review by stevvooe — APPROVED** [review]

**Review by mikebrow — APPROVED** [review]
LGTM
